src/enforce/rules/helpers/find_array_values_in_object_keys/index.js

```diff
@@ -0,0 +1,21 @@
+function findArrayValuesInObjectKyes(array, object) {
+    let foundElements = 0;
+
+    for (let index = 0; index < array.length; index++) {
```

Comment:
Why not just use Array `every` prototype function?
If you don't want ES6 code, I think the following is more readable:
```
function findArrayValuesInObjectKyes(array, object) {
  let foundElements = 0;
  let i=0;

  while (object.hasOwnproperty(array[i])) {
    foundElements++;
    i++;
  }

  return foundElements === array.length;
}
```